websites/J/JavaScript/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "Vyrek",
+      "id": "538421122920742942"
+    },
+    "url": "developer.mozilla.org",
+    "description": {
+      "es": "Pone en tu presencia lo que estas viendo en los docs JS",
+      "en": "Show the things that are you seeing in the JS docs"
+    },
+    "service": "JavaScript | MDN",
+    "version": "1.0.0",
+    "logo": "https://vyrekxd.is-inside.me/jNHWfjeY.png",
+    "thumbnail": "https://vyrekxd.is-inside.me/hkWnnOcf.png",
```

Comment:
upload to imgur

Reply:
Yes let me change it, i cant because i need my phone and i lost it LMAO